h3/bq/sql/H3_DISTANCE.sql

```diff
@@ -0,0 +1,33 @@
+-----------------------------------------------------------------------
+--
+-- Copyright (C) 2021 CARTO
+--
+-----------------------------------------------------------------------
+
+CREATE OR REPLACE FUNCTION `@@BQ_PROJECTID@@.@@BQ_DATASET_H3@@.__H3_DISTANCE`(index_lower1 INT64, index_upper1 INT64, index_lower2 INT64, index_upper2 INT64)
+    RETURNS INT64
+    DETERMINISTIC
+    LANGUAGE js
+    OPTIONS (library=["@@H3_BQ_LIBRARY@@"])
+AS
+"""
+    if (index_lower1 == null || index_upper1 == null || index_lower2 == null || index_upper2 == null)
+        return null;
+    const index1 = [Number(index_lower1), Number(index_upper1)];
```

Comment:
Some functions are checking if H3 inputs are valid and some not.
Maybe add here the check for both inputs.

Reply:
This is because internally the module is already handling it. There are integration tests checking it just in case.